flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX <= 0)
```

Comment:
Maybe we should stick to only `== 0`. Something like `new FlxGame(-10, -50, PlayState)` would result in windowSize x windowHeight, which seems a little weird.